### PR TITLE
Attempt to fix global search disappearing options. Also fixes console…

### PR DESCRIPTION
**Pull request by sri2k1us**
… warning about `li` being a child of `li`

### CODE REVIEW

File: src/components/search/GlobalSearchField.js [hunk 1]
```diff
@@ -402,78 +407,81 @@ function GlobalSearchField(props) {
...
-        analysesI18n,
     ]);
 
+    useEffect(() => {
+        let timeOutId = null;
+        if (searchTerm && searchTerm.length > 2) {
+            timeOutId = setTimeout(() => doSearch(), 500);
+        }
+        return () => clearTimeout(timeOutId);
+    }, [searchTerm, doSearch]);
```
Comment (sri2k1us) [inline src/components/search/GlobalSearchField.js:483]: add .5s delay to begin search after user stops typing.

File: src/components/search/GlobalSearchField.js [hunk 2]
```diff
@@ -118,34 +120,35 @@ const SearchOption = React.forwardRef((props, ref) => {
...
+                        wrap={true}
+                        color={theme.palette.info.primary}
+                    >
+                        <Highlighter search={searchTerm}>
+                            {secondary}
+                        </Highlighter>
+                    </Typography>
+                </div>
+            </MuiLink>
+        </>
```
Comment (sri2k1us) [inline src/components/search/GlobalSearchField.js:151]: Custom option renderer to eliminate the console warning -DOM validation - <li> cannot be child of <li>

**Comment by sri2k1us** [comment]
So I figured this issue behind disappearing search results - polling for session validity. And this wasn’t happening in local dev cos I had the polling timer set to some insanely high number in local config :facepalm:

Regardless, the changes in this PR is still good.
 - Start search only after user has stopped typing
- Fix the console warnings.

**Review by johnworth — APPROVED** [review]
LGTM :shipit:

**Review by slr71 — APPROVED** [review]
I didn't catch any problems. :shipit:

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**